Getting started / Use degit example has '&' instead of '&&'

<!--
Thanks for raising an issue! (For *questions*, we recommend instead using https://stackoverflow.com and adding the 'svelte' tag.)

To help us help you, if you've found a bug please consider the following:

* If you can demonstrate the bug using https://svelte.dev/repl, please do.
* If that's not possible, we recommend creating a small repo that illustrates the problem.
* Make sure you include information about the browser, and which version of Svelte you're using

Reproductions should be small, self-contained, correct examples – http://sscce.org.

Occasionally, this won't be possible, and that's fine – we still appreciate you raising the issue. But please understand that Svelte is run by unpaid volunteers in their free time, and issues that follow these instructions will get fixed faster.

If you have a stack trace to include, we recommend putting inside a `<details>` block for the sake of the thread's readability:

<details>
  <summary>Stack trace</summary>

  Stack trace goes here...
</details>
-->
This is a documentation error and probably also an error in the `npm run dev` code when it finds port 5000 is already in use.

### 1) Documentation error
In the Getting started page [here](https://svelte.dev/blog/the-easiest-way-to-get-started), under the heading "Use degit" the example features a single apersand, which spins off `npm run dev` in the background. This will cause problems if the process isn't killed manually. So:

```
npm run dev & open http://localhost:5000
```
should be:
```
npm run dev && open http://localhost:5000
```

### 2) Possible error in `npm run dev` code.

If you do `npm run dev` when port 5000 is already in use (e.g. because you copied and pasted the getting started command (`npm run dev & open http://localhost:5000`) the result is confusing and will take time to figure out.

To replicate this, do:
```
$ npm run dev &
```
And then:
```
$ npm run dev

> svelte-app@1.0.0 dev /home/mrh/src/svelte/svapp
> run-p start:dev autobuild


> svelte-app@1.0.0 start:dev /home/mrh/src/svelte/svapp
> sirv public --dev


> svelte-app@1.0.0 autobuild /home/mrh/src/svelte/svapp
> rollup -c -w



  Your application is ready~! 🚀

  - Local:      http://localhost:5000
  - Network:    http://192.168.43.137:5000

────────────────── LOGS ──────────────────

{ Error: listen EADDRINUSE :::35729
    at Object._errnoException (util.js:992:11)
    at _exceptionWithHostPort (util.js:1014:20)
    at Server.setupListenHandle [as _listen2] (net.js:1355:14)
    at listenInCluster (net.js:1396:12)
    at Server.listen (net.js:1480:7)
    at Server.listen (/home/mrh/src/svelte/svapp/node_modules/livereload/lib/livereload.js:80:28)
    at Object.exports.createServer (/home/mrh/src/svelte/svapp/node_modules/livereload/lib/livereload.js:246:14)
    at livereload$1 (/home/mrh/src/svelte/svapp/node_modules/rollup-plugin-livereload/dist/index.cjs.js:19:27)
    at Object.<anonymous> (/home/mrh/src/svelte/svapp/rollup.config.js:44:18)
    at Module._compile (module.js:652:30)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'listen',
  address: '::',
  port: 35729 }
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! svelte-app@1.0.0 autobuild: `rollup -c -w`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the svelte-app@1.0.0 autobuild script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/mrh/.npm/_logs/2019-05-17T16_22_47_694Z-debug.log
ERROR: "autobuild" exited with 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! svelte-app@1.0.0 dev: `run-p start:dev autobuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the svelte-app@1.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/mrh/.npm/_logs/2019-05-17T16_22_47_722Z-debug.log
```

--- a/site/content/blog/2017-08-07-the-easiest-way-to-get-started.md
+++ b/site/content/blog/2017-08-07-the-easiest-way-to-get-started.md
@@ -44,8 +44,10 @@ In the terminal, you can instantly create a new project like so:
 npx degit sveltejs/template my-svelte-project
 cd my-svelte-project
 npm install
-npm run dev && open http://localhost:5000
+npm run dev
 ```
+
+This will create a new project in the `my-svelte-project` directory, install its dependencies, and start a server on http://localhost:5000.
 
 Once you've tinkered a bit and understood how everything fits together, you can fork [sveltejs/template](https://github.com/sveltejs/template) and start doing this instead:
 

--- a/site/content/blog/2017-08-07-the-easiest-way-to-get-started.md
+++ b/site/content/blog/2017-08-07-the-easiest-way-to-get-started.md
@@ -44,7 +44,7 @@ In the terminal, you can instantly create a new project like so:
 npx degit sveltejs/template my-svelte-project
 cd my-svelte-project
 npm install
-npm run dev & open http://localhost:5000
+npm run dev && open http://localhost:5000
 ```
 
 Once you've tinkered a bit and understood how everything fits together, you can fork [sveltejs/template](https://github.com/sveltejs/template) and start doing this instead:

